docker/Dockerfile

```diff
@@ -1,10 +1,14 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.19.13-4.1697647145
+FROM registry.access.redhat.com/ubi9/go-toolset:1.22
```

Comment:
Decided to use the `1.22` tag here to avoid potential breaking changes with future go versions.